packages/web/src/builders/dev-server/schema.json

```diff
@@ -52,6 +52,16 @@
     "allowedHosts": {
       "type": "string",
       "description": "This option allows you to whitelist services that are allowed to access the dev server."
+    },
+    "memoryLimit": {
+      "type": "number",
+      "description": "Memory limit for type checking service process in MB.",
+      "default": 2048
+    },
+    "maxWorkers": {
+      "type": "number",
+      "description": "Number of workers to use for type checking.",
+      "default": "# of CPUS-2"
```

Comment:
Since we are using `serveOptions.maxWorkers || ForkTsCheckerWebpackPlugin.TWO_CPUS_FREE`, I don't think we need the default value here

Reply:
Thanks, You're right.